plugin/src/main/java/org/opensearch/ml/processor/MLInferenceSearchResponseProcessor.java

```diff
@@ -316,22 +321,42 @@ private void rewriteResponseDocuments(SearchResponse response, ActionListener<Se
      * @param inputMapIndex           the index of the input mapping to process
      * @param batchPredictionListener the listener to be notified when the predictions are processed
      * @param hitCountInPredictions   a map to keep track of the count of hits that have the required input fields for each round of prediction
+     * @param queryString
```

Comment:
minor: add some explanation

Reply:
add explanation in commit  
[add example in javadoc](https://github.com/opensearch-project/ml-commons/pull/2899/commits/98dfbec04af434462193653fe07f0ce6d8435df7)